system_tests/local/test_plugin.py

```diff
@@ -37,7 +33,12 @@
 
 class TestWorkflowClean(EC2LocalTestUtils):
 
+    def tearDown(self):
+        super(TestWorkflowClean, self).tearDown()
+        self.localenv.execute('uninstall', task_retries=10)
```

Comment:
assert that after the uninstall these operations are still valid. (instance_id or security group is not there anymore)
Then it's no longer part of the teardown. it has some assertions